SPECIFICATION.md

```diff
@@ -118,46 +117,50 @@ Each sprite is a collection of cells or "sprite parts", small textures taken fro
-| u16    | Texture X      | X position of the image in the texture sheet (top-left corner) |
-| u16    | Texture Y      | Y position of the image in the texture sheet (top-left corner) |
-| u32    | Unknown        | ?                                                              |
-| u16    | Texture width  | Width of the image in the texture sheet                        |
-| u16    | Texture height | Height of the image in the texture sheet                       |
-| u16    | X position     | X position of the image in the sprite                          |
-| u16    | Y position     | Y position of the image in the sprite                          |
-| f32    | Scale (X axis) | Scaling factor for the X axis                                  |
-| f32    | Scale (Y axis) | Scaling factor for the Y axis                                  |
-| f32    | Rotation       | Rotation of the sprite part (in degrees)                       |
-| bool   | Flip X         | If true, flip part on X axis                                   |
-| bool   | Flip Y         | If true, flip part on Y axis                                   |
-| u8     | Opacity        | Global opacity for the sprite part                             |
-| 1 byte | Padding        | Referred to as "terminator" on Bread                           |
+| u16    | Region X       | X position of the part's region (top-left corner)              |
+| u16    | Region Y       | Y position of the part's region (top-left corner)              |
+| u16    | Region W       | Width of the part's region                                     |
+| u16    | Region H       | Height of the part's region                                    |
+| u16    | (Reserved)     | Reserved, set by the game                                      |
+| u16    | (Padding)      | 16-bit padding                                                 |
+| s16    | Position X     | The part's X position                                          |
+| s16    | Position Y     | The part's Y position                                          |
+| f32    | Scale X        | Scaling factor for the X axis                                  |
+| f32    | Scale Y        | Scaling factor for the Y axis                                  |
+| f32    | Angle          | Rotation angle in degrees                                      |
+| bool   | Flip X         | If true, flip part on X axis from the center                   |
+| bool   | Flip Y         | If true, flip part on Y axis from the center                   |
+| u8     | Opacity        | The part's opacity                                             |
+| u8     | (Padding)      | 8-bit padding                                                  |
 
 ### Animation:
 
-Each animation is a collection of animation frames, which are references to sprites with specific
+Each animation is a collection of animation keys, which are references to sprites with specific
```

Comment:
any "correctness" reason to use key rather than frame, or is it just for consistency with toast?

Reply:
keys fits better since a key can be extended over multiple frames